ckanext/data_qld/templates/datarequests/close.html

```diff
@@ -2,10 +2,12 @@
 {#
 Handle either old Pylons-style global variables, or newer template parameters.
 #}
-{% set datasets = datasets or c.datasets %}
-{% set datarequest = datarequest or c.datarequest %}
-{% set errors = errors or c.errors %}
-{% set errors_summary = errors_summary or c.errors_summary %}
+{% if datarequest is not defined %}
```

Comment:
why is datarequest the controller of this?

Reply:
It's a good indicator of whether we're running a version of `ckanext-datarequests` that relies on `c` or uses `extra_vars`. The other option would be to individually test for the presence of each parameter, but that seems unnecessary when this is a temporary shim.